### ISSUE TITLE
Hallucinations can occasionally cause vicious continuous client crashes

### ISSUE THREAD

**Issue by aphielanis** [yogstation13/yogstation#1795]
It seems like occasionally severe hallucinations can  cause the client to crash continuously every 10 to 15 seconds. This happened to Silvaplex512 in round 7723 and happened to me a few weeks back. Silva isn't sure why he was hallucinating but I suspect it was changling sting or brain damage and that's how my character started hallucinating too but I am not sure of which.

**Comment by 00aquir** [comment]
yep, in different shifts the hallucinations (singu chasing people) broke the client both times at one point.

**Comment by 00aquir** [comment]
Reference https://github.com/yogstation13/yogstation/issues/519
